Formula/aircrack-ng.rb

```diff
@@ -77,3 +79,26 @@ def caveats
  
  if [ ! -d "${OUI_PATH}" ]; then
  	mkdir -p ${OUI_PATH}
+
+diff --git a/include/aircrack-ng/ce-wpa/pseudo_intrinsics.h b/include/aircrack-ng/ce-wpa/pseudo_intrinsics.h
```

Comment:
What's the source of this patch? Has it been upstreamed?

Reply:
It has not been upstreamed.
I played with it until I got it to compile, then I found I compared it to John the ripper and they have the same patch.

I'm working on getting approval to upstream this patch, but that could take some time.

Reply:
We need this patch upstreamed first, I'm afraid.

Reply:
That's fine.  I'm working on getting it to be upstreamed, but it might take me a really long time.

I found this issue after I had come up with this patch: https://github.com/aircrack-ng/aircrack-ng/issues/1957#issuecomment-834814596

Which led me to:
https://github.com/openwall/john/commit/c9825e688d1fb9fdd8942ceb0a6b4457b0f9f9b4
Which is basically the same thing, just that John the ripper uses v32 and v64, while aircrack doesn't.


We can close this until the patch is upstreamed if you'd like.

Reply:
```
Error: An exception occurred within a child process:
  Errno::ENOENT: No such file or directory @ rb_file_s_rename - (version, version.txt)
```

Almost to the out of tree Building